game.js

```diff
@@ -1959,8 +1959,14 @@ const gameManager = {
     localPlayer.removeAction('activate');
   },
   setAvatarQuality(quality) {
+    metaversefileApi.setQualitySetting(quality);
     const localPlayer = metaversefileApi.useLocalPlayer();
+    const npcs = metaversefileApi.useNpcs()
     localPlayer.avatar.setQuality(quality);
+    npcs.forEach((npc) => {
```

Comment:
IMO we shouldn't iterate the NPCs, but the VRM apps in the `world.appManager`. NPCs are an abstraction over the VRM apps and would automatically be included.

That way even non-NPC VRMs will be optimized.

Reply:
got it, this was based on my assumptions regarding NPC's which I commented earlier, and that the setQuality function was part of the Avatar class.  Will be changing to call the vrm apps's directly.